### PR TITLE
issue: 4153438 DRP use geoDNS registry url

### PR DESCRIPTION
**Pull request by vialogi**
## Description
Use a single registry URL for both production and DR release jobs
Jira: HPCINFRA-2754

##### What
Change harbor registry URL to a single geoDNS URL

##### Why ?
For DRP, use one configuration yaml instead of 2 files.

##### How ?
Use a single URL address.

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

**Comment by vialogi** [comment]
Test release job in  DR: https://nbupdc.blsm.nvidia.com/swx-media-master/job/libvma/job/LibVMA-release-dr/6/
Test release job in PROD: https://nbuprod.blsm.nvidia.com/swx-media-master/job/libvma/job/LibVMA-release/12/

**Comment by vialogi** [comment]
bot:retest

File: .ci/pipeline/release_jjb.yaml [hunk 1]
```diff
@@ -70,6 +70,7 @@
             keep-system-variables: true
             properties-content: |
               jjb_proj={jjb_proj}-dr
+              WEBREPO_URL=webrepo.mtr.labs.mlnx
```
Comment (dpressle) [inline .ci/pipeline/release_jjb.yaml:73]: Are you sure we dont need this for prod job definition as well? did you tested both ?

**Comment by dpressle** [comment]
> Test release job in DR: https://nbupdc.blsm.nvidia.com/swx-media-master/job/libvma/job/LibVMA-release-dr/6/ Test release job in PROD: https://nbuprod.blsm.nvidia.com/swx-media-master/job/libvma/job/LibVMA-release/12/

Your DR test is not good, it did not built the image so you didnt really tested both harbor and webrepo changes

**Pull request closed without merging**